packages/docs-theme/src/styles/_examples.scss

```diff
@@ -32,3 +85,38 @@
     margin-right: 0;
   }
 }
+
+.view-example-source {
+  display: inline-block;
+  margin-bottom: $pt-grid-size * 2;
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  .#{$ns}-icon {
+    margin-right: $pt-grid-size / 2;
+  }
+}
+
+
+[data-modifier] > code {
```

Comment:
i think these styles aren't used now?

Reply:
I think you're right, the ones from the code snippets are handled elsewhere